packages/sourmash/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: sourmash
+  version: 4.7.0
+  top-level:
+    - sourmash
+requirements:
+  run:
+    - screed
+    - cffi
+    - deprecation
+    - cachetools
+    - numpy
+    - matplotlib
+    - scipy
+    - sqlite3
+    - bitstring
+source:
+  #  url: https://pypi.io/packages/source/s/sourmash/sourmash-4.7.0.tar.gz
+  #  sha256: 4135ac82059da24a30f3b9554ecd0107dbf68eac2828f39fa275038554b76311
+  url: https://codeload.github.com/sourmash-bio/sourmash/tar.gz/fecfc97c3ec0be1dcfe43c73a12297f1295b5199
```

Comment:
Yup, will do! I left this PR as draft because the [pyodide-fixing PR](https://github.com/sourmash-bio/sourmash/pull/2433) and CI checks were added after `4.7.0`, but I will release `4.7.1` and fix this URL/sha256

Reply:
[sourmash 4.8.0](https://pypi.org/project/sourmash/4.8.0/) just released, I updated the PR to point to PyPI sdist